.eslintrc

```diff
@@ -14,5 +14,8 @@
     "import/no-unresolved": "off",
     "import/no-unassigned-import": "off",
     "eol-last": "off", // the crowdin stuff doesn't include a line ending 🙄
+  },
+  "globals": {
+    "Prism": false,
```

Comment:
Weird things happened with highlighting if I tried to import Prism, so I'll just use the global :(